src/ot_orchestration/dags/config/genetics_etl_platform.yaml

```diff
@@ -0,0 +1,283 @@
+#
+# Gentropy inputs:
+# gs://genetics_etl_python_playground/input/l2g/gold_standard/curation.json
+# gs://gwas_catalog_top_hits/study_index
+# gs://gwas_catalog_sumstats_susie/study_index
+# gs://gwas_catalog_sumstats_pics/study_index
+# gs://eqtl_catalogue_data/study_index
+# gs://ukb_ppp_eur_data/study_index
+# gs://finngen_data/r11/study_index
+# gs://gwas_catalog_top_hits/credible_sets/
+# gs://gwas_catalog_sumstats_pics/credible_sets/
+# gs://gwas_catalog_sumstats_susie/credible_set_clean/
+# gs://eqtl_catalogue_data/credible_set_datasets/eqtl_catalogue_susie/
+# s://ukb_ppp_eur_data/credible_set_clean/
+# gs://finngen_data/r11/credible_set_datasets/susie/
+# gs://genetics_etl_python_playground/vep/cache (VEP CACHE)
+# gs://genetics_etl_python_playground/static_assets/gnomad_variants
+#
+# Gentropy Outputs:
+# gs://open-targets-pre-data-releases/${release}/output/genetics/parquet/gene_index
+# gs://open-targets-pre-data-releases/${release}/output/genetics/parquet/biosample_index
+# gs://open-targets-pre-data-releases/${release}/output/genetics/parquet/study_index
+# gs://open-targets-pre-data-releases/${release}/output/genetics/parquet/invalid_study_index
+# gs://open-targets-pre-data-releases/${release}/output/genetics/parquet/credible_set
+# gs://open-targets-pre-data-releases/${release}/output/genetics/parquet/invalid_credible_set
+# gs://open-targets-pre-data-releases/${release}/output/genetics/parquet/partitioned_variants
+# gs://open-targets-pre-data-releases/${release}/output/genetics/json/annotated_variants
+# gs://open-targets-pre-data-releases/${release}/output/genetics/parquet/variant_index
+# gs://open-targets-pre-data-releases/${release}/output/genetics/parquet/colocalisation/coloc
+# gs://open-targets-pre-data-releases/${release}/output/genetics/parquet/colocalisation/ecaviar
+# gs://open-targets-pre-data-releases/${release}/output/genetics/parquet/l2g_feature_matrix
+# gs://open-targets-pre-data-releases/${release}/output/genetics/models/locus_to_gene_model/classifier.skops
+# gs://open-targets-pre-data-releases/${release}/output/genetics/parquet/l2g_predictions
+# gs://open-targets-ppp-releases/${release}/input/evidence-files/l2g.json.gz
+
+release: 24.12
+dataproc:
+  python_main_module: gs://genetics_etl_python_playground/initialisation/gentropy/dev/cli.py
+  cluster_metadata:
+    PACKAGE: gs://genetics_etl_python_playground/initialisation/gentropy/dev/gentropy-0.0.0-py3-none-any.whl
+  cluster_init_script: gs://genetics_etl_python_playground/initialisation/gentropy/dev/install_dependencies_on_cluster.sh
+  cluster_name: otg-etl
+  autoscaling_policy: otg-efm
+  allow_efm: true
+  num_workers: 10
+
+nodes:
+  - id: genetics_gene_index
+    kind: Task
+    prerequisites:
+      - etl_target # ETL step required
+    params:
+      step: gene_index
+      step.target_path: gs://open-targets-pre-data-releases/${release}/output/etl/parquet/targets
```

Comment:
This is not looking good to me. At the end, in the same bucket there will be two datasets named as `targets` and`target_index`. I find it quite confusing. Do we really need both datasets? What is the difference? I know, this question is beyond the scope of this PR.